SUBMODULE_VERSIONS

```diff
@@ -13,7 +13,7 @@ e59c55d78f1a041e6a14771254f8e6280804b430 third_party/ruy
 a1390ed39ec77ecfb574bc6fcd5bfc5e3adbdea9 third_party/sdl2
 685f86471e9d26b3eb7676695a2e2cefb4551ae9 third_party/spirv_cross
 f8bf11a0253a32375c32cad92c841237b96696c0 third_party/spirv_headers
-57eb48aed36160c4876bc8310d9ca84d42ee9e2a third_party/swiftshader
+84f5eeb6dd9b225f465f93737fa76aad7de355cf third_party/swiftshader
```

Comment:
If we're going through the effort to update this, can you bump this again, at least past https://github.com/google/swiftshader/commit/df17a76102dfabb3f1bd6e51449cece9f77b45e3 ? That might let us remove some workarounds like
https://github.com/google/iree/blob/8bda4d9a1241c7716ac6ed0f9738ff57a5e3c25d/iree/hal/cts/cts_test_base.h#L33-L42

Reply:
Yes, this is later then df17a76.